v22.1/changefeed-examples.md

```diff
@@ -399,6 +399,13 @@ You'll need access to a [Google Cloud Project](https://cloud.google.com/resource
     cat key.json | base64
     ~~~
 
+    If you're working on Linux, run the following to ensure that lines are not wrapped in the output:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    cat key.json | base64 -w 0
+    ~~~
+
```

Comment:
nit: I see this more of an either/or. The Linux command doesn't work on Mac, and the Mac command results in output that breaks the `CREATE CHANGEFEED...` command on Linux. I think we can safely avoid by making this something like:

> Next, base64 encode your credentials key using the following command specific to your platform:
>
> If you're working on Linux, run:
>    ~~~ shell
>   cat key.json | base64 -w 0
>    ~~~
>
> If you're working on macOS, run:
>    ~~~ shell
>   cat key.json | base64
>    ~~~

Otherwise LGTM!